hpx/runtime/threads/policies/thread_queue.hpp

```diff
@@ -1128,7 +1152,6 @@ namespace hpx { namespace threads { namespace policies
         std::size_t max_count_;     // maximum number of existing HPX-threads
         task_items_type new_tasks_; // list of new tasks to run
 
-        std::atomic<std::int64_t> new_tasks_count_; // count of new tasks to run
 #ifdef HPX_HAVE_THREAD_QUEUE_WAITTIME
```

Comment:
Hmm, what makes `new_tasks_count_` special? Compared to `thread_map_count_`, work_items_count_` and `terminated_items_count_`.

Reply:
The timings around that variable were sticking out like a sore thumb. The others might need special handling as well - that will require more investigations, though.